Formula/spotbugs.rb

```diff
@@ -0,0 +1,44 @@
+class Spotbugs < Formula
+  desc "Tool for Java static analysis (FindBugs's successor)"
+  homepage "https://spotbugs.github.io/"
+  url "https://repo.maven.apache.org/maven2/com/github/spotbugs/spotbugs/3.1.0/spotbugs-3.1.0.tgz"
+  sha256 "c1eab4dc50cd9854c83fc4a90b4a9b9c63e5307bd0f0f7bb04720e17153b629c"
+
+  head do
+    url "https://github.com/spotbugs/spotbugs.git"
```

Comment:
need a blank line

Reply:
between `url` and `depends_on` inside of the `head` block? sorry, i modeled after `libp11.rb`

Reply:
yes between url and depends_on.

Reply:
Sorry about that, done